### DISCUSSION TITLE
Technical advice

### DISCUSSION THREAD

**Discussion by 18950219762** [graphieros/vue-data-ui#84] [Q&A] [answered]
hello, https://vue-data-ui.graphieros.com On the home page of this address, what frame is used to move the mouse to display the coordinates?

**Comment by graphieros** [comment, marked as answer]
Hello @18950219762

This is actually a component available in Vue Data UI : [VueUiCursor](https://vue-data-ui.graphieros.com/docs#vue-ui-cursor)
